bundles/org.openhab.core/src/main/java/org/openhab/core/library/types/HSBType.java

```diff
@@ -274,7 +275,7 @@ public PercentType[] toRGB() {
         BigDecimal h = hue.divide(BIG_DECIMAL_HUNDRED, 10, RoundingMode.HALF_UP);
         BigDecimal s = saturation.divide(BIG_DECIMAL_HUNDRED);
 
-        int hInt = h.multiply(BigDecimal.valueOf(5)).divide(BigDecimal.valueOf(3), 10, RoundingMode.HALF_UP).intValue();
+        int hInt = h.multiply(BigDecimal.valueOf(5)).divide(BigDecimal.valueOf(3), 0, RoundingMode.DOWN).intValue();
```

Comment:
Using 10 decimal places and the dropping all decimal places afterwards can be replaced by using 0 ddecimal places and roundmode DOWN